File: service/SurfWebAutofac/Model/Dtos/Maps/MapListDto.cs
namespace Model.Dtos.Maps;

public class MapListDto
{
    /// <summary>
    ///     地图ID
    /// </summary>
    public  long Id { get; set; }

    /// <summary>
    ///     名称
    /// </summary>
    public  string Name { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public  string Difficulty { get; set; }

    /// <summary>
    ///     地图图片
    /// </summary>
    public  string Img { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/maps/MapListDto.java
package com.surfweb.system.dto.maps;

import lombok.Data;

@Data
public class MapListDto {

    /**
     * 地图ID
     */
    private long id;

    /**
     * 名称
     */
    private String name;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 地图图片
     */
    private String img;
}
